presto-main/src/main/java/io/prestosql/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -460,7 +459,7 @@ protected Node visitShowCreate(ShowCreate node, Void context)
                     throw semanticException(TABLE_NOT_FOUND, node, "Table '%s' does not exist", objectName);
                 }
 
-                accessControl.checkCanShowColumnsMetadata(session.toSecurityContext(), new CatalogSchemaTableName(tableHandle.get().getCatalogName().getCatalogName(), objectName.asSchemaTableName()));
```

Comment:
I think we should remove this method like https://github.com/prestosql/presto/pull/2899. Users should be allowed to display list of columns that they are allowed to query.